tests/test_client.py

```diff
@@ -224,6 +234,29 @@ def test_check_input_parameters_beta(model_client, va_governor_config):
             fixed_effects,
             pi_method,
             "bad_beta",
+            winsorize,
+            robust,
+            lambda_,
+            handle_unreporting,
+        )
+
+
+def test_check_input_parameters_winsorize(model_client, va_config):
```

Comment:
This unit test is failing for me. Can you try to merge develop into this branch?